watcher/watcher.go

```diff
@@ -51,9 +135,40 @@ func NewWatchHandler(ctx context.Context) (*WatchHandler, error) {
 	wh.buildMaps(ctx, podsList)
 
 	wh.currentPodListResourceVersion = podsList.GetResourceVersion()
+
+	wh.startCleanerRoutine(ctx)
+
 	return wh, nil
 }
 
+// start routine which cleans up unused imageIDs and instanceIDs from storage, and triggers relevancy scan
+func (wh *WatchHandler) startCleanerRoutine(ctx context.Context) {
```

Comment:
[TODO]: Consider renaming this function because you are calling here the `triggerRelevancyScan` function.